cli/tests/unit/utime_test.ts

```diff
@@ -1,18 +1,11 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
 import {
   unitTest,
-  assert,
+  assertEquals,
   assertThrows,
   assertThrowsAsync,
 } from "./test_util.ts";
 
-// Allow 10 second difference.
-// Note this might not be enough for FAT (but we are not testing on such fs).
```

Comment:
Nothing (symlinks etc) work on a FAT filesystem so shouldn't be concerned about that.